lib/puppet/provider/gluster_volume/gluster_volume.rb

```diff
@@ -4,7 +4,7 @@
   :gluster_volume, :parent => Puppet::Provider::Gluster) do
 
   has_command(:gluster, 'gluster') do
-    environment :HOME => "/tmp"
+    environment :HOME => '/tmp'
```

Comment:
This should be the last file. We still need `""` for places where we're evaluating expressions inside the string, though.